include/ddns/cloudflare-ddns.h

```diff
@@ -24,88 +24,88 @@
 extern "C" {
 #endif
 
-#define TACHI_ZONE_ID_LENGTH         32U
-#define TACHI_RECORD_ID_LENGTH       32U
-#define TACHI_RECORD_NAME_MAX_LENGTH 255U
-#define TACHI_IP_ADDRESS_MAX_LENGTH  46U
+#define DDNS_ZONE_ID_LENGTH         32U
```

Comment:
Just a suggestion, I find "DDNS" a bit generic, if you would go with "CFDDNS" (CF for Cloudflare) there is less chance of a clash.

Reply:
Yes, DDNS by itself is generic, but I thought that the actual chances of collision are quite low... When would you need to use a library that updates DNS records dynamically using Cloudflare's API and at the same time use some other kind of library that defines macros or functions named the same way?

CFDDNS or CDDNS could absolutely be used, but I like four characters long namespaces more :)

Reply:
Right, that is indeed very unlikely.

Reply:
At the same time, if I expand this tool to make use of let's say Google's API it would make sense to name these symbols differently. I would still use "ddns" as a common namespace for general things (like IP_ADDRESS_MAX_LENGTH), but put API-specific macros and functions in their respective namespace, like "ddns_c" and "ddns_g". I'll think about this a bit more.

Reply:
> Edit: I'll keep things simple for now. After all, the project is called "cloudflare-ddns", not "any-ddns"

I was exactly about to point this out - if you were to extend this further to other services, you would need a source rename and probably a SOVERSION bump anyway.